ocp4/profiles/cis.profile

```diff
@@ -157,7 +157,14 @@ selections:
     - kubelet_configure_tls_key
     # Like kubelet_disable_readonly_port but check for .apiServerArguments["kubelet-client-key"]
   # 4.2.13 Ensure that the Kubelet only makes use of Strong Cryptographic Ciphers
+    - kubelet_configure_tls_cipher_suites_ingresscontroller
+    - kubelet_configure_tls_cipher_suites_kubeapiserver_operator
+    - kubelet_configure_tls_cipher_suites_openshift_authentication
```

Comment:
This rule seems to be missing and it's failing when building the image because of it:

```
 ValueError: Rule 'kubelet_configure_tls_cipher_suites_openshift_authentication' was not found in the benchmark. Please remove rule 'kubelet_configure_tls_cipher_suites_openshift_authentication' from profile 'cis' before proceeding. 
```